src/Core/Components/Pagination/FluentPaginator.razor.cs

```diff
@@ -81,6 +79,8 @@ protected override void OnParametersSet()
     /// <inheritdoc />
     public void Dispose()
```

Comment:
Can you use/override the `FluentComponentBase.DisposeAsync`, to centralize all "disposing process" in the base classes.